ext/MitsubaLoader/CMitsubaMaterialCompilerFrontend.cpp

```diff
@@ -244,44 +244,30 @@ namespace MitsubaLoader
         case CElementBSDF::THINDIELECTRIC:
         case CElementBSDF::ROUGHDIELECTRIC:
         {
-            nextSym = ir->allocNode<IR::CBSDFCombinerNode>(IR::CBSDFCombinerNode::ET_FRESNEL_BLEND);
-            nextSym->children.count = 2u;
+            auto* dielectric = ir->allocNode<IR::CDielectricBSDFNode>();
+            nextSym = dielectric;
 
             const float eta = current->dielectric.intIOR/current->dielectric.extIOR;
             _IRR_DEBUG_BREAK_IF(eta==1.f);
             if (eta==1.f)
                 os::Printer::log("WARNING: Dielectric with IoR=1.0!", current->id, ELL_ERROR);
 
-            auto& refl = nextSym->children[0];
-            refl = ir->allocNode<IR::CMicrofacetSpecularBSDFNode>();
-            auto& trans = nextSym->children[1];
-            trans = ir->allocNode<IR::CMicrofacetSpecularBSDFNode>();
-
-            auto* node_refl = static_cast<IR::CMicrofacetSpecularBSDFNode*>(refl);
-            auto* node_trans = static_cast<IR::CMicrofacetSpecularBSDFNode*>(trans);
-
-            node_refl->scatteringMode = IR::CMicrofacetSpecularBSDFNode::ESM_REFLECT;
-            node_refl->shadowing = IR::CMicrofacetSpecularBSDFNode::EST_SMITH;
-            node_refl->eta = IR::INode::color_t(eta);
+            dielectric->scatteringMode = IR::CMicrofacetSpecularBSDFNode::ESM_REFLECT;
```

Comment:
why is it a reflect now?

Reply:
i think i can get rid of this scattering mode actually
`REFLECT` is set without a reason, it's ignored by GLSL backend anyway